package.json

```diff
@@ -68,6 +67,8 @@
     "chart.js": "3.7.0",
     "checklist-model": "1.0.0",
     "ckeditor4": "4.17.1",
+    "cqm-execution": "^3.1.0",
+    "cqm-models": "^3.1.0",
```

Comment:
just confirming that these package.json changes are supposed to be here.

Reply:
yes but they should be locked to version. Will fix in my PR #5239

Reply:
I think these version specs are supposed to be in the oe-cqm-service package.json. This may be duplicated there, so the lines in OpenEMR’s package.json can be removed. I installed them via command-line during development.

Reply:
So OpenEMR would require oe-cqm-service, and oe-cqm-service package.json would have these dependencies listed.